src/main/java/unibook/ui/UiManager.java

```diff
@@ -90,6 +90,15 @@ public boolean isModuleListShowing() {
         return mainWindow.isModuleListShowing();
     }
 
+
```

Comment:
Should we add java docs for this portion? Not 100% sure if it's needed but it's a public method that is not a getter/setter.
```
/**
* Sets the UI to show Module List view.
*/
```